src/AutorestSwift/View Models/OperationViewModel.swift

```diff
@@ -114,9 +111,9 @@ struct Params {
     // Whether to 'var' or 'let' in generated code for the param declaration
     var declaration: String = "var"
 
-    init(from params: Params? = nil) {
-        self.required = params?.required ?? [KeyValueViewModel]()
-        self.optional = params?.optional ?? [KeyValueViewModel]()
+    init() {
+        self.required = [KeyValueViewModel]()
+        self.optional = [KeyValueViewModel]()
     }
```

Comment:
Why was this removed?

Reply:
There is nowhere in the code which we pass in param when create this struct. So I remove the code to simplify the code.